tests/subsys/nrf_security/psa_core_lite/testcase.yaml

```
@@ -0,0 +1,107 @@
+tests:
+  # PSA core lite: Ed25519
+  psa_core_lite.ed25519:
+    sysbuild: true
+    platform_allow: >
+      nrf54l15dk/nrf54l15/cpuapp
```

`platform_allow` must be a list:
```suggestion
    platform_allow:
      - nrf54l15dk/nrf54l15/cpuapp
```

---

This may have not been removed in NCS but it's deprecated and for example has been removed in upstream Zephyr and I know there have been efforts to migrate away from space lists in NCS.

---

Fixed